libs/blocks/iframe/iframe.js

```diff
@@ -0,0 +1,13 @@
+export default function init(a) {
+  const url = a?.href ?? a?.querySelector('a')?.href;
```

Comment:
I would suggest renaming `a` to be `link`. It seems like there's a scenario where the arg could be a div with an anchor in it. I think if the param name is a tag name, it really should only accept that tag.

Reply:
The convention seems to be `a` for autoBlocks and `el` for blocks, since this works for both `link` seems like a good option.